networkd/network_server.ml

```diff
@@ -844,6 +844,35 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				let ifaces = List.filter (fun iface ->
+					String.startswith "eth" iface) (Ovs.bridge_to_interfaces name) in
```

Comment:
Don't use the device name to determine physicalness. This may turn out unreliable. Use `Sysfs.is_physical`.